IEEE Xplore.js

```diff
@@ -516,10 +528,9 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"date": "January 1991",
+				"date": "一月 1991",
```

Comment:
I assume this is a localization based on your IP address. If so, we'd want to keep it set to the English version.

Reply:
OK, I change it from `一月` to `January` manually because `一月` has exactly the same meaning as `January`. You are right. Geographic location of my IP address is China. So when I click `update` in Scaffold, it change `January` to `一月`. Sorry, I wasn't noticing this.